tests/atomistics/structure/test_atoms.py

```diff
@@ -1607,6 +1607,14 @@ def test_get_wrapped_coordinates(self):
             np.linalg.norm(position-structure.cell*0.1), 0
         )
 
+    def test_is_skewed(self):
```

Comment:
I think this would easier to read if `is_skewed` is called with an explicit tolerance.

Reply:
Done!

Reply:
I put a message. Was this more or less what you suggested?

Reply:
Perfect, thanks!